akka-docs/src/test/java/jdocs/stream/IntegrationDocTest.java

```diff
@@ -752,8 +752,8 @@ public void illustrateSourceQueue() throws Exception {
     new TestKit(system) {
       {
         // #source-queue
-        int bufferSize = 5;
-        int elementsToProcess = 3;
+        int bufferSize = 10;
```

Comment:
Context: In [Source queue](https://doc.akka.io/docs/akka/current/stream/operators/Source/queue.html) documentation page, documentation is not inline to example. So corrected the example. 

As documentation says, `elementsToProcess` changed it from `3` to `5`. To avoid confusion changed `bufferSize` as well  

```
In combination with the queue, the throttle operator can be used to control the processing to a given limit, e.g. 5 elements per 3 seconds.
```

Reply:
Ok, I see, thanks for the clarification.